var/spack/repos/builtin/packages/py-pyspoa/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyPyspoa(PythonPackage):
+    """Python bindings to spoa"""
+
+    homepage = "https://github.com/nanoporetech/pyspoa"
+    pypi = "pyspoa/pyspoa-0.0.8.tar.gz"
+
+    version("0.0.8", sha256="8299d18066b498a6ef294c5a33a99266ded06eeb022f67488d2caecba974b0a4")
+
+    depends_on("py-wheel@0.34.1:", type="build")
```

Comment:
Where did you find this wheel version restriction?

Reply:
https://github.com/nanoporetech/pyspoa/blob/v0.0.8/requirements.txt

Reply:
This file isn't included in the source distribution, I wouldn't trust it.